master/components/dashboard/js/modules/controllers/cAdvisorController.js

```diff
@@ -22,7 +22,11 @@ app.controller('cAdvisorController', [
         $scope.minions = res;
 
         var promises = lodash.map(res.items, function(m) {
-            return cAdvisorService.getDataForMinion(m.metadata.name, (m.status.conditions[0].status === 'True'));
+            var readyCondition = m.status.conditions.filter(function ( obj ) {
+                return obj.type === 'Ready';
+            })[0];
```

Comment:
I can check for that condition and default to no if it's not found, wasn't sure if that status was always returned or not.